swagger.yaml

```diff
@@ -284,52 +363,58 @@ paths:
           description: "Failed to list all versions of user secret"
   /secret/version_list_for_link:
     post:
-      description: "List user secrets versions for link for authorized request"
+      description: "Lists information about all versions of a secret. The secret is identified by a key name that contains a link."
```

Comment:
for lines longer than 80 characters, please use the `>` option in YAML.

To learn more: https://stackoverflow.com/a/3790497